scripts/plugins/Health.js

```diff
@@ -12,16 +12,23 @@ CivClicker.plugins.Health = (() => {
      *
      */
     constructor() {
-      this.temperature = 0;
+      this.temperature = 30; // start at a warm temp
+      this.isNight = false; // start at day
     }
 
     /**
      * Tick.
      */
     tick() {
-      // Check temperature
-      // Check heat in society.
-      // If night, cold and rainy, make someone sick possibly.
+      /*
+      / Check temperature
+      / Check heat in society.
+      / If night, cold and rainy, make someone sick possibly.
+      */
+      let chance = Math.floor(Math.random() * 1000) + 1; // 0.1% chance
+      if(this.temperature < 15 /* CELSIUS - if Farinheit then < 60*/ && (chance === 1000 || this.isNight)) {
```

Comment:
15 C is your limit? :D You must be from a warm place! In Sweden where I come from 15 is considered summer, haha.

But more seriously, we should compare towards the temperature of the "civilization" - meaning weather + hearths, air conditioning, what-have-you - and not the weather temperature directly. I think we could create a registry plugin to gather common properties like that. (Registry is also a design pattern for a big, global object.) Then this plugin can fetch data from the registry, and the registry will separately calculate all temperature issues into one value.

Reply:
A registry is a good idea. It can hold temperature and things like that.

[SIDENOTE] the `wolf-plugin` branch

Reply:
&nbsp;&nbsp;&nbsp;&nbsp;So I've been thinking – 15° C is the limit for now because of a reason – the workers are *naked*. This means that 15° C is cold for them. If you live in a rural area you can try this for yourself by going outside naked when it's 15° C - it's going to be **cold**.

Maybe you can add clothing just like you added axes/spears?

- [ ] Add Clothing?

Reply:
Sounds good. Preferably, 15 degrees should be a setting inside config.json (read on startup).